modules/op-pipelines-as-code-splitting-logs-by-namespace.adoc

```diff
@@ -0,0 +1,14 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: REFERENCE
+[id="pipelines-as-code-splitting-logs-by-namespace_{context}"]
+= {pac} splitting logs by namespace
+
+The logs contain the namespace information to make it possible to grep logs on namespace or split the logs by a particular namespace. For example, to view the logs and search for `namespace` information, enter the following command:
```

Comment:
to view the logs related to the namespace `mynamespace`, enter the following command:
`$ oc logs pipelines-as-code-controller-59c9d5db68-fkp7n -n openshift-pipelines | grep mynamespace`
@savitaashture @piyush-garg WDYT?

Reply:
I think the `mynamespace` makes more sense here. Thank you! The answer to the `openshift-pipelines` namespace is under this comment: https://github.com/openshift/openshift-docs/pull/60644#discussion_r1217664689

Reply:
yes providing with example as